src/Tizen.Multimedia.MediaPlayer/Player/SphericalVideo.cs

```diff
@@ -0,0 +1,329 @@
+        /// <since_tizen> 5 </since_tizen>
+        public bool IsSphericalContent()
+        {
+            Player.ValidatePlayerState(PlayerState.Ready, PlayerState.Playing, PlayerState.Paused);
+
+            NativePlayer.IsSphericalContent(Player.Handle, out var value).
+                ThrowIfFailed(Player, "Failed to get the spherical state of the content");
+
+            return value;
+        }
+
+        /// <summary>
+        /// Gets the level of the zoom when it is spherical.
+        /// </summary>
+        /// <remarks>Remark.</remarks>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <exception cref="InvalidOperationException">The player is not in the valid state.</exception>
+        /// <seealso cref="SetZoom(float)"/>
+        /// <since_tizen> 5 </since_tizen>
+        public float GetZoom()
+        {
+            Player.ValidateNotDisposed();
+
+            NativePlayer.GetZoom(Player.Handle, out var value).
+                ThrowIfFailed(Player, "Failed to get the level of the zoom");
+
+            return value;
+        }
+
+        /// <summary>
+        /// Gets the level of the zoom when it is spherical .
+        /// </summary>
+        /// <remarks>Remark.</remarks>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <exception cref="InvalidOperationException">The player is not in the valid state.</exception>
+        /// <seealso cref="GetZoom()"/>
+        /// <since_tizen> 5 </since_tizen>
+        public void SetZoom(float level)
+        {
+            Player.ValidateNotDisposed();
+
+            if (level < 1.0F || 10.0F < level)
+            {
+                throw new ArgumentOutOfRangeException(nameof(level), level, "Valid level is 1.0 to 10.0");
+            }
+
+            NativePlayer.SetZoom(Player.Handle, level).ThrowIfFailed(Player, "Failed to set the level of the zoom.");
+        }
+
+        /// <summary>
+        /// Gets or sets the spherical mode.
+        /// </summary>
+        /// <remarks>The player must be in the <see cref="PlayerState.Ready"/>, <see cref="PlayerState.Playing"/>,
+        /// or <see cref="PlayerState.Paused"/> state.</remarks>
+        /// <exception cref="ObjectDisposedException">The player has already been disposed of.</exception>
+        /// <exception cref="InvalidOperationException">The player is not in the valid state.</exception>
+        /// <since_tizen> 5 </since_tizen>
+        public bool IsSphericalMode
+        {
+            get
+            {
+                NativePlayer.IsSphericalMode(Player.Handle, out var value).
+                    ThrowIfFailed(Player, "Failed to get the spherical mode of the player");
+                return value;
+            }
+
+            set
+            {
+                Player.ValidateNotDisposed();
```

Comment:
Don't we need to check this in ```get``` accessor?

Reply:
i think all properties in player should be considered about this issue